fastestimator/trace/io/best_model_saver.py

```diff
@@ -45,10 +45,11 @@ def __init__(self,
             assert hasattr(model, "loss_name"), "cannot infer model loss name, please put the model to UpdateOp first"
             assert len(model.loss_name) == 1, "the model has more than one losses, please provide the metric explicitly"
             metric = next(iter(model.loss_name))
-        super().__init__(mode="eval", inputs=metric)
+        super().__init__(mode="eval", inputs=metric, outputs=["since_best", "{}_{}".format(save_best_mode, metric)])
```

Comment:
it does not make sense to move this here. I might want to know when my best model was without having to save the best model.

Reply:
resolved.